### PR TITLE
Feature/measure to reg

### PR DESCRIPTION
**Pull request by cqc-melf**
Not sure what the name is er agreed on, happy to update this.

### CODE REVIEW

File: pytket/qir/conversion/conversion.py [hunk 1]
```diff
@@ -202,6 +202,19 @@ def __init__(
             ),
         )
 
+        # void mz_to_reg(qubit, i1* creg, int)
```
Comment (qartik) [inline pytket/qir/conversion/conversion.py]: Could you write the signature here in full? What's the `int` type referring to?

**Review by qartik — CHANGES_REQUESTED** [review]

**Comment by qartik** [comment]
> Not sure what the name is er agreed on

Given the interface in https://github.com/CQCL/pytket-qir/pull/61#pullrequestreview-1567770240 perhaps a better name might be `measure_to_creg` or `mz_to_creg` in short?

File: pytket/qir/conversion/conversion.py (hunk 1 above)
Comment (cqc-melf) [reply pytket/qir/conversion/conversion.py]: done

**Comment by cqc-melf** [comment]
> > Not sure what the name is er agreed on
> 
> Given the interface in [#61 (review)](https://github.com/CQCL/pytket-qir/pull/61#pullrequestreview-1567770240) perhaps a better name might be `measure_to_creg` or `mz_to_creg` in short?

I have done a rename to `mz_to_creg`

**Comment by qartik** [comment]
Hi @cqc-melf, it's not clear to me from a quick look whether `mz_to_creg` is supposed to measure a single qubit or a register of qubits or both. Can you clarify?

If it's a single bit (that I suspect from the third argument of creg index), then the name could be misleading. In that case, I'd prefer it be called `mz_to_creg_bit`.

**Comment by cqc-melf** [comment]
Yes the plan is to measure one qubit to the bit entry of a register, I have added an additional comment and renamed like suggested

**Review by qartik — COMMENTED** [review]
LGTM, but I will let @peter-campora chip in as well before approval.

**Review by cqc-alec — APPROVED** [review]

**Review by peter-campora — APPROVED** [review]
`mz_to_creg_bit` is a name that works for me. Will add support for it in L3 soon!

**Review by qartik — APPROVED** [review]

**Pull request merged**